mkdocs.yml

```diff
@@ -149,6 +149,8 @@ pages:
         - Twilio: integration/alerts-twilio-integration.md
         - VictorOps: integration/alerts-victorops-integration.md
         - Zapier: integration/alerts-zapier-integration.md
+    - Orchestration:
+        - Terraform: orchestration/terraform.md
```

Comment:
+1 for having Integration > Orchestration in the left nav, but shouldn't the MD file live under docs/integrations/ ? Integrations that we list there are not just things we monitor, but anything we integrate with.  e.g. we have Slack there.

Reply:
Sure, makes sense. We can do that as well. Have Integration > Orchestration > Terraform. Then we can have more Orchestration docs under that dir. :+1:

Reply:
Shifted.